### PR TITLE
Material UI Changes

### PR DESCRIPTION
**Pull request by Kdecherf**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | -
| Documentation | no
| Translation   | no
| Fixed tickets | -
| License       | MIT

Following comments in #2618, this PR includes several changes I made on my side to the Material theme.

I split changes into small commits to be able to easily drop irrelevant ones.

Feel free to comment or tell which one to keep, I will push a final commit with the updated minified style before the merge (_if it's accepted_).

Here are some side-by-side screenshots to compare:

left: original / right: changes
2016-11-20-205301_1920x2160
2016-11-20-205254_1920x2160
2016-11-20-205241_1920x2160

### CODE REVIEW

**Comment by nicosomb** [comment]
I like the new typo. I will look on my computer tomorrow.

**Comment by tcitworld** [comment]
Remarks : 
- font ✔️ 
- links : why not but please keep a little color difference too. You must be able to spot them while browsing the text fast
- titles font size : could it be increased a little bit ?
- `<code>` ✔️ 

Please run `npm i` and `grunt tests` to check your changes and build minified files.

**Comment by Kdecherf** [comment]
Another screenshot, featuring changes for ul elements:

2016-11-20-222820_1920x2160

**Comment by Kdecherf** [comment]
@tcitworld I updated title sizes and put back the original color for links.

Also, I normalized all changed colors according to the color palette of Google Material.

Here is an up-to-date screenshot (without h1 and h2):

2016-11-20-233956_1920x2160

File: app/Resources/static/themes/material/css/main.css [hunk 1]
```diff
@@ -596,6 +645,47 @@ a.original {
...
+#article article blockquote {
+  font-style: italic;
+}
+
+#article article strong {
+  font-weight: bold;
+}
+
+#article article pre {
+  -moz-box-sizing: border-box;
```
Comment (tcitworld) [inline app/Resources/static/themes/material/css/main.css]: No need for this `-moz` prefix since Firefox 29, I guess we don't need it anymore.

**Comment by nicosomb** [comment]
Can you change the target branch of your PR (-> 2.2) please?

**Comment by j0k3r** [comment]
And I suggest that you can squash your commits before rebased your branch against the 2.2, you'll have less conflict to resolve :)

**Comment by Kdecherf** [comment]
As far as the only reported conflict is `style.min.css` I prefer to not squash commits :)

I will remove the moz stuff and resolve the conflict this evening.

**Comment by Kdecherf** [comment]
Well, the rebase was more painful than I expected.

Anyway, two things before the merge:

- Please keep in mind that the new font-family is `Georgia, serif` and Georgia is not embedded in the project. The browser will fallback to the default serif font if Georgia is missing (_which should be quite rare now_).
- I noticed something suspect during the rebase of the minified style file: a line completely unrelated to my changes was modified (_check .mdi-rotate-270_) and I don't know why.

**Comment by Kdecherf** [comment]
As an alternative to Georgia we could embed `Noto Serif`.

**Comment by tcitworld** [comment]
> Please keep in mind that the new font-family is Georgia, serif and Georgia is not embedded in the project. The browser will fallback to the default serif font if Georgia is missing (which should be quite rare now).

Not trying to be picky here but afaik Georgia is a Microsoft font and subject to [this EULA](https://web.archive.org/web/20020227054122/www.microsoft.com/typography/fontpack/eula.htm). Therefore, not included by default on nearly all GNU/Linux distributions.

**Comment by nicosomb** [comment]
So "we could embed Noto Serif" 👍

**Comment by Kdecherf** [comment]
Ack, I will make the change this evening :)

**Comment by zertrin** [comment]
Chiming in late, but I have some remarks:

- First of all, is the new design an optional choice? or does it replace the current theme with no possibility to keep it as it was?
- Some studies consider sans-serif fonts more readable, and I personally think that is the case. If you think that you prefer serif fonts I won't argue with your preferences, but please let the user have the choice and don't impose your preferences to everyone.

**Comment by zertrin** [comment]
Chiming in late, but I have some remarks:

- First of all, is the new design an optional choice? or does it replace the current theme with no possibility to keep it as it was?
- Some studies consider sans-serif fonts more readable, and I personally think that is the case. If you think that you prefer serif fonts I won't argue with your preferences, but please let the user have the choice and don't impose your preferences to everyone.

PS: Readability had the possibility to switch between serif or sans-serif as well as 4 color themes IIRC. Maybe a wishlist item for Wallabag. In any case, choosing between serif and sans-serif is a much bigger deal than the color choices.

**Comment by tcitworld** [comment]
> First of all, is the new design an optional choice?

No. I found the changes to be minimal.

Please open a new issue for the serif/non-serif functionality.

**Comment by nicosomb** [comment]
We talk about serif / sans serif here https://github.com/wallabag/wallabag/issues/1802

**Comment by zertrin** [comment]
Minimal?!?? The difference between serif/sans-serif is all but minimal, it's the first thing you notice since that's a change that affects all the text content, which is at the core of Wallabag (read it later (implied comfortably)).

Please don't impose a serif font to users accustomed to the current design. You may add the choice of a serif font, maybe by default if you really love it, but please don't remove sans-serif because of a sudden change of taste from a contributor.

**Comment by tcitworld** [comment]
I was talking about the other changes, not the serif/non-serif change.

**Comment by Kdecherf** [comment]
I am okay to introduce a change in this PR to let the user choose between serif/sans-serif but how do we handle the font-family regarding the minified file?

**Comment by nicosomb** [comment]
@Kdecherf Please remove the font change from this PR. We need to discuss how to implement a font selector in an other PR or in #1802. 
We are OK for the other changes, so we can merge this PR when font will be removed.

**Comment by Kdecherf** [comment]
@nicosomb OK, I will remove the font part this evening and work on a new PR for that.

**Comment by Kdecherf** [comment]
The PR has been updated to drop the font family change.

Here is a side-by-side screenshot of changes between the last two revisions:
2016-11-25-135006_1920x2160

And here is a side-by-side screenshot of changes between 2.2 and this PR at this current state:
2016-11-25-135013_1920x2160

**Comment by nicosomb** [comment]
(I really like the font on the left side)

Could you fix the alignment of the icons at the top of the article? Or do we need to open a new PR?

**Comment by Kdecherf** [comment]
I dropped the updated minified file because I am unable to reproduce the correct changes before the pull request (_the grunt output on top of 2.2 is different between my machine and the repository_).

So someone (_the last who changed the file_) must update this PR with the updated file.

**Comment by nicosomb** [comment]
Will do.

**Comment by Kdecherf** [comment]
@nicosomb the serif font has been dropped regarding the comment of @zertrin.

I will introduce it in a new PR regarding #1802 and #1278

**Comment by Kdecherf** [comment]
I'm checking for alignment of icons

**Comment by Kdecherf** [comment]
@nicosomb it appears that icons alignment has been fixed by 34aa06a9b9c9bc6cdfb61607b2def8e1fc861b76 but it was not ported to 2.2

**Comment by nicosomb** [comment]
@j0k3r any opinion about that? You merged master in 2.2 since 16 october.

**Comment by nicosomb** [comment]
Thank you @Kdecherf!

**Pull request merged**